arelle/plugin/validate/EDINET/rules/upload.py

```diff
@@ -676,13 +684,15 @@ def rule_html_uris(
                 yield Validation.error(
                     codes='EDINET.EC1014E',
                     msg=_("The URI in the HTML specifies a path to a file that doesn't exist. "
-                          "File name: %(file)s (line %(line)s). "
+                          "File name: '%(file)s' (line %(line)s). "
                           "Please update the URI to reference a file."),
                     file=doc.basename,
                     line=elt.sourceline,
                     modelObject=elt,
                 )
                 continue
+            fileSourcePath = fullPath.relative_to(basePath)
+            pluginData.addUsedFilepath(val.modelXbrl, fileSourcePath)
```

Comment:
It seems somewhat fragile for the other issue's logic to depend on this rule running.  I wonder if the processing of the HTML URIs should be stored in plugin data so this issue and the other can work off the same?